### PR TITLE
fix(decide): Ignore conflicts for racy updates

### PR DESCRIPTION
**Pull request by neilkakkar**
## Changes

Fixes https://sentry.io/organizations/posthog/issues/3653986251/events/latest/?project=1899813&referrer=latest-event#request

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

I spent quite a bit of time trying to make a race condition test, but this is hard to do reliably 😅

### CODE REVIEW

File: posthog/test/test_feature_flag.py [hunk 1]
```diff
@@ -1681,6 +1685,69 @@ def test_entire_flow_with_hash_key_override(self):
...
+            key="multivariate-flag",
+            created_by=user,
+            ensure_experience_continuity=True,
+        )
+
+        Person.objects.create(
+            team=team, distinct_ids=["example_id"], properties={"email": "tim@posthog.com", "team": "posthog"}
+        )
+
+        import concurrent.futures
```
Comment (hazzadous) [inline posthog/test/test_feature_flag.py:1731]: Can we move this import to the top of the file?

Comment (neilkakkar) [reply posthog/test/test_feature_flag.py]: if you ask so nicely, yes

**Review by hazzadous — APPROVED** [review]
Looks like it should deal with the immediate issue 👍 

Non-blocking aside: does the test consistently fail without this fix?

**Comment by neilkakkar** [comment]
In the 10 tries I did, it failed all 10 times, so I'm happy; but I can imagine there being cases where it doesn't

**Pull request merged**